ports/chromaprint/vcpkg.json

```
@@ -1,6 +1,7 @@
 {
   "name": "chromaprint",
   "version-string": "1.5.0",
```

```suggestion
  "version": "1.5.0",
```
We might use [version](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.reference.md#version) here.